modules/mvt/test/mvt-loader.spec.js

```diff
@@ -93,7 +93,7 @@ test('Polygons MVT to GeoJSON', async t => {
   const loaderOptions = {
     mvt: {
       geojson: true,
-      _tileProperties: {
+      tileIndex: {
```

Comment:
Nits:
- Add test cases for bad option combos (geojson: true, no tileIndex, bad tileIndex).
- If throwing, I recommend adding `t.throws(() => func)` test cases.